Portfolio: setup and mobile first

Khaled-AlKhateeb opened the pull request:
tasks I did

-adjusted linter files on the directory.
-fixed issues regarding linter on the index.html file.
-fixed issues regarding linter on the style.css file.
-fixed all the errors after running tests.
-edited HTML elements to match the HTML semantic tags.
-adjusted some CSS properties to maintain the style according to the HTML changes.

ichala reviewed (changes requested):
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @Khaled-AlKhateeb 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### 1. You are required to use the correct Linters setup [CSS and HTML linters](https://github.com/microverseinc/linters-config/tree/master/html-css) 
### 2. After setting up the correct linters, kindly make sure they are all green before requesting another review 
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me @ichala in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

Khaled-AlKhateeb commented:
Please review my code for any issues.

sja-thedude reviewed (changes requested):
## Invalid Code Review Request ❌

Hi @Khaled-AlKhateeb, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that is for Portfolio: finish mobile version , kindly make sure to submit for Portfolio: Setup.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Khaled-AlKhateeb commented:
**tasks I did** 

-adjusted linter files on the directory.
-fixed issues regarding linter on the index.html file
-fixed issues regarding linter on the style.css file 
-fixed all the errors after running tests.

HAMMAS-SALEEM commented on index.html:
```diff
@@ -1,13 +1,68 @@
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <nav class="toolBar">
```
### Some spacing on right side
![horiazontal-scroll (copy)](https://user-images.githubusercontent.com/93039359/185389781-1c585868-1440-4a3e-b7b6-043b6d5b69ef.png)

- [ ] Please remove this spacing on the left side to make your design look professional and to follow the Figma template.
### **Quick Fix**
**Step 1**
![quick-fix (copy)](https://user-images.githubusercontent.com/93039359/185389712-bc34cef0-abeb-46d0-bac8-8bc58c245b5f.png)

**Step 2**
![quick-fix2 (copy)](https://user-images.githubusercontent.com/93039359/185389744-97ac4b94-26a4-49e3-a26d-68121afe76fc.png)

frank1738 reviewed (approved):
Hi @Khaled-AlKhateeb , 

Good work implementing changes requested by the previous reviewers 👏
 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![5SW](https://user-images.githubusercontent.com/77038610/154919611-7d0675af-6364-45a3-92d6-eccd9b743ced.gif)

## Highlights

- Have used the right GitHub flow ✅
- Meaningful title and summary for PR ✅
- Professional README.md file ✅
- Linter checks passed ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
